src/types/ScholarshipAmount.ts

```diff
@@ -23,9 +23,12 @@ export default class ScholarshipAmount {
     type: AmountType = AmountType.Unknown,
     data?: { min?: number; max?: number }
   ) {
-    this.type = type;
     const [min, max] = [data?.min, data?.max];
 
+    if (type === AmountType.Varies && !min && !max)
+      this.type = AmountType.Unknown;
+    else this.type = type;
```

Comment:
```suggestion
    this.type = type === AmountType.Varies && !min && !max ? AmountType.Unknown : type;
```

Reply:
Lol I had this at first but changed it cuz I thought the other one was more legible.